### DISCUSSION TITLE
DFC Crate help required

### DISCUSSION THREAD

**Discussion by Kirill6453** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1152] [Q&A]
Hello, please tell me how to create items using DFC Crate

**Comment by BallOfEnergy1** [comment]
A bit late yes, but you use a normal DFC setup, and then instead of a receiver, you put a tungsten crate. You also have to make sure the spark quantities are sufficient for that recipe; I'm not sure on the exact numbers right now, but it should be enough to just run the DFC on a decent fuel.
